model/src/main/scala/explore/model/ObsSummary.scala

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats._
+import explore.model.enum.ObsStatus
+import io.circe.Decoder
+import io.circe.Encoder
+import io.circe.HCursor
+import io.circe.Json
+import io.circe.syntax._
+import lucuma.core.model.Asterism
+import lucuma.core.model.Observation
+import lucuma.core.model.Target
+import monocle.macros.Lenses
+
+import java.time.Duration
+import java.time.temporal.ChronoUnit
+
+@Lenses
+final case class ObsSummary(
+  id:                Observation.Id,
+  name:              Option[String],
+  status:            ObsStatus = ObsStatus.New,
+  conf:              String = "GMOS-N R831 1x300",
+  constraints:       String = "<0.7\" <0.3 mag Bright",
+  duration:          Duration = Duration.of(93, ChronoUnit.MINUTES),
+  observationTarget: Option[Either[Asterism.Id, Target.Id]]
+)
+
+object ObsSummary {
+  implicit val eqObsSummary: Eq[ObsSummary] = Eq.by(x => (x.id, x.name, x.observationTarget))
+
+  implicit val obsSummaryEncoder: Encoder[ObsSummary] = new Encoder[ObsSummary] {
+    final def apply(c: ObsSummary): Json = {
+      val common = Json.obj(("id", c.id.asJson), ("name", c.name.asJson))
```

Comment:
I think we can unify the logic a bit (and avoid intermediate allocations and the `deepMerge`) with something like:

``` scala
  val common = JsonObject(...)
  (c.observationTarget match {
    case None => common
    case Some(t) => common.add("observationTarget", t match {
      case Right(tid) => Json.obj("target_id" -> tid.asJson)
      case Left(aid) => Json.obj("asterism_id" -> aid.asJson)
    })
  }).asJson
```

Would it work if we add to `common` something like: `"observationTarget" ->